specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/examples/SessionPools_Patch.json

```diff
@@ -0,0 +1,95 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "rg",
+    "sessionPoolName": "testsessionpool0",
+    "api-version": "2024-02-02-preview",
+    "sessionPoolEnvelope": {
+      "properties": {
+        "scaleConfiguration": {
+          "maxConcurrentSessions": 500,
+          "readySessionInstances": 100
+        },
+        "dynamicPoolConfiguration": {
+          "executionType": "Timed",
+          "cooldownPeriodInSeconds": 600
+        },
+        "customContainerConfiguration": {
+          "containers": [
+            {
+              "image": "repo/testcontainer:v4",
+              "name": "testinitcontainer",
+              "resources": {
+                "cpu": 0.2,
+                "memory": "100Mi"
+              },
+              "command": [
+                "/bin/sh"
+              ],
+              "args": [
+                "-c",
+                "while true; do echo hello; sleep 10;done"
+              ]
+            }
+          ],
+          "ingress": {
+            "targetPort": 80
+          }
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/rg/providers/Microsoft.App/sessionPools/testsessionpool",
+        "name": "testsessionpool",
+        "type": "Microsoft.App/sessionPools",
+        "location": "East US",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "environmentId": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/rg/providers/Microsoft.App/managedEnvironments/demokube",
+          "poolManagementType": "Dynamic",
+          "containerType": "CustomContainer",
+          "scaleConfiguration": {
+            "maxConcurrentSessions": 500,
+            "readySessionInstances": 100
+          },
+          "dynamicPoolConfiguration": {
+            "executionType": "Timed",
+            "cooldownPeriodInSeconds": 600
+          },
+          "customContainerConfiguration": {
+            "containers": [
+              {
+                "image": "repo/testcontainer:v4",
+                "name": "testinitcontainer",
+                "resources": {
+                  "cpu": 0.2,
+                  "memory": "100Mi"
+                },
+                "command": [
+                  "/bin/sh"
+                ],
+                "args": [
+                  "-c",
+                  "while true; do echo hello; sleep 10;done"
+                ]
+              }
+            ],
+            "ingress": {
+              "targetPort": 80
+            }
+          },
+          "poolManagementEndpoint": "https://testsessionpool.agreeableriver-3d30edf1.eastus.azurecontainerapps.io"
+        }
+      }
+    },
+    "202": {
```

Comment:
>202 [](http://example.com/codeflow?start=5&length=3)

Do we need to return the payload body, according to [https://github.com/cloud-and-ai-microsoft/resource-provider-contract/blob/master/v1.0/resource-api-reference.md#patch-resource](https://github.com/cloud-and-ai-microsoft/resource-provider-contract/blob/master/v1.0/resource-api-reference.md#patch-resource)?

Reply:
I think yes, but I referenced other resources like managedEnvironment, container app, container app job, and they don't have body for patch 202 response, so I followed this pattern for now.